src/App.js

```diff
@@ -110,7 +112,7 @@ class App extends React.PureComponent<Props, State> {
   };
 
   renderApp() {
-    const route = this.props.route;
+    const { route, emergencies, language, onClose } = this.props;
```

Comment:
In the [MR in gitlab](https://gitlab.skypicker.com/frontend/frontend/merge_requests/5695/diffs#9df85bf500d7d18a4217b99ce1d61e635fbd2479_63_75) you're calling this prop `banners` and here `emeregencies`

Reply:
thanks, last minute updates 🤦‍♂️   fixed on FE